components/dashboard/src/data/prebuilds/prebuild-queries.ts

```diff
@@ -49,9 +49,7 @@ export function useTriggerPrebuildQuery(configurationId?: string, gitRef?: strin
             if (!configurationId) {
                 throw new ApplicationError(ErrorCodes.BAD_REQUEST, "prebuild configurationId is missing");
             }
-            if (!gitRef) {
```

Comment:
When we don't provide a ref, we run on the default branch. This is how we did it in the previous project UIs.

Reply:
Project UI allows to start a prebuild for different branch
<img width="1516" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/8c00fc94-8e05-4e40-909a-004a451b2bd9">